Source/Core/DolphinQt/Settings/USBDeviceAddToWhitelistDialog.cpp

```diff
@@ -174,3 +180,41 @@ void USBDeviceAddToWhitelistDialog::OnDeviceSelection()
   device_vid_textbox->setText(*vid);
   device_pid_textbox->setText(*pid);
 }
+
+void USBDeviceAddToWhitelistDialog::OnVIDTextboxTextChanged()
```

Comment:
Is this the same as OnPIDTextboxTextChanged? Could the two be combined into one function?